### PR TITLE
Changes:solution and program creation api for samiksha added

### PR DESCRIPTION
**Pull request by praveenKDass**

### CODE REVIEW

File: src/constants/config.json [hunk 1]
```diff
@@ -4407,6 +4407,19 @@
 				}
 			]
 		},
+		{
```
Comment (VISHNUDAS-tunerlabs) [inline src/constants/config.json:4410]: @praveenKDass for api with :id we might need config without id also. I have noticed it please check once

File: src/constants/config.json [hunk 2]
```diff
@@ -4485,6 +4498,305 @@
...
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "samiksha",
+					"packageName": "elevate-survey"
+				}
+			]
+		},
+		{
+			"sourceRoute": "/samiksha/v1/solutions/isTargetedBasedOnUserProfile/:id",
```
Comment (VISHNUDAS-tunerlabs) [inline src/constants/config.json:4593]: @praveenKDass if api accept req body the method should be post, please check this for all apis you added

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]

File: src/constants/config.json (hunk 2 above)
Comment (praveenKDass) [reply src/constants/config.json:4671]: Changed as post method @VISHNUDAS-tunerlabs

File: src/constants/config.json (hunk 1 above)
Comment (praveenKDass) [reply src/constants/config.json:4410]: Added without id @VISHNUDAS-tunerlabs

**Pull request merged**